docker/docker-compose-observability.yml

```diff
@@ -1,13 +1,17 @@
 version: "3"
 
 volumes:
+  grafana_data: {}
   prometheus_data: {}
 
 services:
   grafana:
     image: grafana/grafana:8.4.3@sha256:44ea09598b05bd6296f663bb728eb6af90f45399648d85b4123bbc3f271a121b
     container_name: grafana
     restart: unless-stopped
+    volumes: 
+      - ./.grafana/provisioning/:/etc/grafana/provisioning/
```

Comment:
Maybe we can store all the tempory data in a path like : docker/.data/{service}

Reply:
Solution is https://github.com/cfbarbero/tick-grafana-docker/issues/1#issuecomment-636894425. So I decide to keep docker volumes